src/Contexts/VersionContext.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import axios from 'axios';
+import logger from '../Utilities/Logger';
+import { version as packageVersion } from '../../package.json';
+import localPreferences from '../Utilities/LocalPreferences';
+import moment from 'moment';
+import _ from 'lodash';
+
+const { versionCheck } = localPreferences;
+
+interface VersionInformation {
+    serverFrontend?: string;
+    currentFrontend: string;
+}
+
+const getVersionFile = async () => {
+    return axios.get<string>(`/version.txt?cache_bust=${new Date().getTime()}`);
+};
+
+
+const versionRefresh = async () => {
+    // To test in development make a version.txt in the public folder with a value that matches or does not match package.json (i.e. "1.5.5")
+    // Then comment out this check
+    // version.txt in production is generated in post build
+    // if (process.env.NODE_ENV !== 'production') {
```

Comment:
We should remove the comment above or uncomment this

When this was actually doing a refresh and logging warnings this was to bypass in dev